app/services/provider_authorisation.rb

```diff
@@ -5,21 +5,25 @@ def initialize(actor:)
     @actor = actor
   end
 
-  def can_make_offer?(application_choice:)
-    @actor.is_a?(SupportUser) || @actor.providers.include?(application_choice.course.provider)
-  end
-
-  def can_change_offer?(application_choice:, course_option_id:)
-    course_option = CourseOption.find course_option_id
+  def can_make_offer?(application_choice:, course_option_id: nil)
+    supplied_course_option = CourseOption.find(course_option_id) if course_option_id
     if @actor.is_a?(SupportUser)
       true
+    elsif supplied_course_option && course_option_id != application_choice.course_option.id
+      application_choice_belongs_to_user_providers?(application_choice: application_choice) && \
```

Comment:
Right, so you can't transfer _off_ a course you don't control. That seems correct.